src/turnkeyml/cli/cli.py

```diff
@@ -90,11 +85,11 @@ def main():
     # Parser for the "benchmark" command
     #######################################
 
-    def check_extension(choices, file_name):
+    def check_extension(choices, file_name, error_func):
         _, extension = os.path.splitext(file_name.split("::")[0])
         if extension[1:].lower() not in choices:
-            raise exceptions.ArgError(
-                f"input_files must end with .py, .onnx, or .txt (got '{file_name}')"
+            error_func(
+                f"input_files must end with .py, .onnx, or .txt (got '{file_name}')\n"
```

Comment:
Allow this function to use the argparse standard error flow (instead of our bespoke exception class)